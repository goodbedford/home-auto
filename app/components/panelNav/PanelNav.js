import React from "react";


const PanelNav = (props) => {

  return (
    <div className="home-auto__panel-rooms home-auto__panel-rooms--column">
      <div className="home-auto__section-group">
        {props.children}
      </div>
    </div>
  );
};

export default PanelNav;
