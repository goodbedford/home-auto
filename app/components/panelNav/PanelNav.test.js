/* eslint-disable no-undef */

import React from "react";
import {shallow} from "enzyme";
import toJson from "enzyme-to-json";
import PanelNav from "./PanelNav";

describe("<PanelNav />", () => {

  it("PanelNav snapshot", () => {
    const wrapper = shallow(<PanelNav />);
    expect(toJson(wrapper)).toMatchSnapshot();
  });
});
