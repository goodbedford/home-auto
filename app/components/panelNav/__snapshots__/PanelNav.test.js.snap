// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<PanelNav /> PanelNav snapshot 1`] = `
<div
  className="home-auto__panel-rooms home-auto__panel-rooms--column"
>
  <div
    className="home-auto__section-group"
  />
</div>
`;
